public/ACMW/css/events.css

```diff
@@ -40,83 +31,62 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
+	background-color: #2d2d30 !important;
 	border: 0;
-	font-size: 11px;
+	font-size: 11px !important;
 	letter-spacing: 4px;
 	opacity: 0.9;
 }
-.navbar li a,
+.navbar li a, 
 .navbar .navbar-brand {
-	color: #d5d5d5;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)